server.yml

```diff
@@ -95,6 +95,21 @@
         job: ipa-backup
         special_time: daily
 
+    - name: manage kerberos log with logrotate
+      copy:
+        dest: /etc/logrotate.d/krb5kdc
+        content: |
+          /var/log/krb5kdc.log{
+            missingok
+            notifempty
+            weekly
+            rotate 12
+            postrotate
+              /bin/kill -HUP `cat /var/run/krb5kdc.pid 2>/dev/null` 2> /dev/null || true
```

Comment:
Please do not send SIGHUP to the daemon directly, use systemd instead
You can check the proper action at /usr/lib/systemd/system/krb5kdc.service